examples/README.md

```diff
@@ -41,6 +41,29 @@ python min_oe_openMM.py --inmols examples/tripos_mol2/ \
 * `SMIRNOFF/` - minimized mol2 files with SMIRNOFF
 * `output.dat` - output file reporting any errors with min_oe_openMM.py
 
+<<<<<<< HEAD
```

Comment:
@nthi0625 
These <<< show up when there are conflicting changes, could you fix this section?